api/desecapi/metrics.py

```diff
@@ -27,7 +27,10 @@ def set(name, *args, **kwargs):
 set('desecapi_key_encryption_success', 'number of times key encryption was successfull')
 set('desecapi_key_decryption_success', 'number of times key decryption was successfull')
 
-#.py metrics
+#exception_handlers.py metrics
+
+set('desecapi_database_unavailability_count', 'number of times database was unavailable')
```

Comment:
`..._count` is not very descriptive (everything counts something). Better: `database_unavailable`